pom.xml

```diff
@@ -49,16 +56,17 @@
     <version.org.kie7.final>7.32.0.Final</version.org.kie7.final>
     <version.org.kie7>7.32.0.Final</version.org.kie7>
     <version.com.thoughtworks.xstream>1.4.11.1</version.com.thoughtworks.xstream>
-    <version.jaxb>2.3.0</version.jaxb>
-    <version.jsonb>1.0</version.jsonb>
-    <version.jsonp>1.1.4</version.jsonp>
+    <version.javax.xml.bind>2.3.0</version.javax.xml.bind>
+    <version.com.sun.xml.bind>2.3.0</version.com.sun.xml.bind>
+    <version.javax.json.bind>1.0</version.javax.json.bind>
+    <version.javax.json>1.1.4</version.javax.json>
     <version.javax.activation>1.2.0</version.javax.activation>
     <version.com.google.protobuf>3.6.1</version.com.google.protobuf>
     <version.org.eclipse.jdt>3.19.0</version.org.eclipse.jdt>
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
     <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
-    <version.kubernetes-client>4.6.2</version.kubernetes-client>
-    <version.okhttp>3.12.6</version.okhttp>
+    <version.io.fabric>4.6.2</version.io.fabric>
```

Comment:
can we add Kubernetes-client here as io.fabric is too generic

Reply:
+1

Reply:
+1

Reply:
i guess we can, I will change/add